app/models.py

```diff
@@ -0,0 +1,106 @@
+
+from sqlalchemy import Integer,BigInteger, Column, Boolean,String,Text,Date,ForeignKey,create_engine
+from sqlalchemy.orm import sessionmaker, relationship, declarative_base
+
+Base = declarative_base()
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id = Column(Integer,primary_key=True,autoincrement=True)
+    telegram_id = Column(BigInteger,unique=True)
+    childrens = relationship('Children',back_populates='user')
+
+    def __repr__(self):
+        return f"{self.telegram_id}"
+    
+class Children(Base):
+    __tablename__ = 'childrens'
+
+    id = Column(Integer,primary_key=True, autoincrement=True)
+    user_id = Column(Integer, ForeignKey('users.id'), nullable=False)
+    name = Column(String, nullable=False)
+    birth_date = Column(Date, nullable=False)
+
+    user = relationship('User',back_populates='childrens')
+    diagnosis_history = relationship("DiagnosisHistory",back_populates='child')
+    diagnosis_result = relationship("DiagnosisResult",back_populates='child')
+
+    def __repr__(self):
+        return f'{self.user_id}, {self.name} {self.birth_date}'
+
+class SkillTypes(Base):
```

Comment:
```suggestion
class SkillType(Base):
```
обычно класс представляет один объект, а таблица в базе данных - множество объектов определенного типа, поэтому в БД  таблицы называются во множественном числе (users, skill_types), а в моделях - в единственном